packages/loader/test-loader-utils/README.md

```diff
@@ -2,14 +2,43 @@
 
 Test utilities for the Fluid Framework Loader. Includes mock implementations of various loader interfaces.
 
-<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README:scripts=FALSE&installation=FALSE&apiDocs=FALSE&help=FALSE) -->
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README_HEADER&apiDocs=FALSE) -->
```

Comment:
Future improvement plan: Default `apiDocs` based on package scope. Disabled (by default) for `@fluid-internal` and `@fluid-private`